projects/components/src/lib/paginator/paginator.component.css

```diff
@@ -0,0 +1,22 @@
+::ng-deep .pagination {
+  display: flex;
+}
+
+::ng-deep .page-link {
+  --tw-text-opacity: 1;
+    color: rgb(107 114 128 / var(--tw-text-opacity));
+    font-weight: 500;
+    font-size: 0.875rem;
+    line-height: 1.25rem;
```

Comment:
Would be better to use tailwind classes to ensure same font size and weight configuration. Avoid ::ng-deep, https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep.